### PR TITLE
add initial server module

### PR DESCRIPTION
**Pull request by slugbyte**
initalized npm package
created server module
 Changes to be committed:
    new file:   package.json
    new file:   server.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "git+https://github.com/slugbyte/express-demo-deploy.git"
+  },
+  "author": "duncan@codefellows.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/slugbyte/express-demo-deploy/issues"
+  },
+  "homepage": "https://github.com/slugbyte/express-demo-deploy#readme",
+  "dependencies": {
+    "express": "^4.14.0",
```
Comment (fraziermork) [inline package.json:20]: hmm....

File: server.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
```
Comment (fraziermork) [inline server.js:1]: nice work!

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "type": "git",
+    "url": "git+https://github.com/slugbyte/express-demo-deploy.git"
+  },
+  "author": "duncan@codefellows.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/slugbyte/express-demo-deploy/issues"
+  },
+  "homepage": "https://github.com/slugbyte/express-demo-deploy#readme",
+  "dependencies": {
```
Comment (slugbyte) [inline package.json:19]: tha slkj flskjdf lksdj lksajf lkjads flkj sdlfkj alfkdj dsalkfj lakdsjf lkajds flkadsfj ladskjf

**Comment by fraziermork** [comment]
done w/ feature

**Pull request merged**